KIPs/kip-81.md

```diff
@@ -0,0 +1,783 @@
+---
+kip: 81
+title: Implementing the on-chain governance voting method
+author: Yeri<yeriel.lee@krustuniverse.com>, Daniel<daniel.cc@krustuniverse.com>, Aidan<aidan.kwon@krustuniverse.com>, Ollie<ollie.j@krustuniverse.com>, Eddie<eddie.kim0@krustuniverse.com>
+discussion-to: https://govforum.klaytn.foundation/t/proposal-implementing-the-on-chain-governance-voting-method/18
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-09-19
+---
+
+
+## Simple Summary
+Introducing a new governance voting method based on the staking amount and implementation of the Klaytn Square, a governance portal
+
+## Abstract
+Klaytn introduces a stake-based governance model that provides voting power to governance participants. Currently, one vote per Governance Council(GC) member was cast. The new method will introduce the voting right that will be exercised based on the staking amount with the maximum cap to prevent an entity from making an arbitrary decision. The voting agenda is determined through discussion on the Klaytn Governance Forum, and the topic will be presented and voted on at the newly launched Governance Portal, Klaytn Square. This voting mechanism aims to provide responsibility and obligation of voting to Governance Councils.
+
+## Motivation
+The new on-chain voting mechanism discloses GC’s opinion transparently through the portal, allowing anyone to view the result. The governance agenda discussed in Klaytn Governance Forum will be voted on the governance portal, Klaytn Square. Since the voting power is calculated based on the staking amount, this voting process provides more power to GC members who share the same interest as Klaytn by staking and locking up more KLAYs.
+
+## Specification
+Klaytn Governance Forum is to freely propose and discuss Klaytn governance items. Once Klaytn Square, the governance portal, opens, the on-chain voting will be executed based on the discussion held in this forum.
+
+Klaytn Square includes the following functions:
+- Ability to vote on a governance agenda and view the voting process
+- Information about Governance Councils: description, contract address, notice, and staking status, staking and reward history
+- View the history of governance agenda and Governance Councils
+
+![voting process diagram](../assets/kip-81/voting_process_diagram.png)
+
+The foundation will provide 7 days of preparation period for voting, providing a time for GC to adjust the staking amount. With the start of the voting, the foundation will announce the list of GC members and their voting power. GC will have 7 days of the voting period.
```

Comment:
How do the foundation announce the list of GC members and their voting power?
through Klaytn Square? Will Klaytn Square have the functionality?

Reply:
Yes, Klaytn Square has the functionality. For each proposal, list of eligible voters and their voting powers are displayed.

Reply:
@blukat29 okay great. Thanks for the answer.